appium/tests/data/index.ts

```diff
@@ -2,7 +2,8 @@ export const CommonData = {
     account: {
         email: process.env.ACCOUNT_EMAIL,
         password: process.env.ACCOUNT_PASSWORD,
-        passPhrase: process.env.PASS_PHRASE
+        passPhrase: process.env.PASS_PHRASE,
+        wrongPassPhrase: 'user is not able to see email'
```

Comment:
`wrongPassPhrase` doesn't have to be here, there's nothing global about it. I recommend to hard code it in the test spec. Also I'd just use `wrong` string since it's faster to type in the test.